pmoves/tools/_secrets_common.py

```
@@ -122,19 +122,86 @@ def local_env_path() -> Path:
         key = key.strip()
-        if key:
-            values[key] = value
+
+        # Validate key format (uppercase alphanumeric with underscores)
+        if not key or not key.isupper() or not key.replace("_", "").isalnum():
+            # Skip lines that don't look like KEY=VALUE format
+            # This catches base64 fragments with '=' padding like "abcd1234=="
+            continue
+
+        # Validate the value format
+        is_valid, error = validate_secret_value(key, value)
```

Ruff (0.15.10)

[warning] Unpacked variable `error` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)